internal/app/secretless/listeners/sshagent/listener.go

```diff
@@ -2,14 +2,14 @@ package sshagent
 
 import (
 	"fmt"
+	"github.com/cyberark/secretless-broker/pkg/secretless/config/v1"
```

Comment:
Ditto moving internal import to bottom of import list.
Ditto on naming this import config_v1